src/components/MdTable/MdTablePagination.vue

```diff
@@ -4,86 +4,133 @@
       <span class="md-table-pagination-label">{{ mdLabel }}</span>
 
       <md-field>
-        <md-select v-model="currentPageSize" md-dense md-class="md-pagination-select" @changed="setPageSize">
+        <md-select v-model="currentPageSize" md-dense md-class="md-pagination-select">
           <md-option v-for="amount in mdPageOptions" :key="amount" :value="amount">{{ amount }}</md-option>
         </md-select>
       </md-field>
     </template>
 
-    <span>{{ currentItemCount }}-{{ currentPageCount }} {{ mdSeparator }} {{ mdTotal }}</span>
+    <span>{{ mdPage }}/{{ pageCount }}</span>
```

Comment:
Any reason why this was changed? Should be following material design spec and display the current range.